docs/get_started/index.md

```diff
@@ -1,2 +1,164 @@
-# Get Started
+Getting Started
+===============
 
+Introduction[](#introduction)
+-----------------------------
+
+i8 terminal is a modern python-based terminal application that gives you superior power and flexibility to understand and analyze the market. The interface is simple, efficient, and powerful: it's command-line!
+
+**i8 Terminal is open-source and backed by the Investoreight Platform and currently covers major U.S. exchanges.**
+
+Table of Contents
+-----------------
+
+*   [Introduction](#introduction)
+*   [Installation](#installation)
+    *   [Binary Installer](#binary-installer)
+    *   [Installing with Python pip](#installing-with-python-pip)
+    *   [Running from Code](#running-from-code)
+*   [How to Run I8 Terminal](#how-to-run-i8-terminal)
+*   [Sign Up / Sign In to the i8 Terminal Server](#signup-signin-to-the-i8-terminal-server)
+*   [Subscription Plans](#subscription-plans)
+*   [Your First Command](#your-first-command)
+
+* * *
+
+Installation[](#installation)
+---------------------------------------------------------------
+
+The I8 Terminal can be directly installed on your computer via our installation program. Within this section, you are guided through the installation process. If you are a developer, please have a look [here](https://https://github.com/investoreight/i8-terminal). If you struggle with the installation process, please visit our [contact page](https://www.i8terminal.io/contact).
```

Comment:
The I8 Terminal => i8 Terminal. Note that you should always use small case i for i8. Also, never use article `the` for i8 Terminal. It's a name and the article is not required.